Iterators-chapter-1.js

```diff
@@ -0,0 +1,43 @@
+/*
+forEach()
+1. Дан массив с числами(сам придумай). Создай новый массив, состоящий из квадратов этих чисел.
+*/
+const numbers = [1, -2, 3, -4, 5, -6];
+const squaredNumbers = [];
+numbers.forEach(numbersItem =>
+    squaredNumbers.push(Math.pow(numbersItem, 2))
+);
+console.log(squaredNumbers);
+/*
+2. Дан массив с числами. Найди сумму этих чисел.
+*/
+let sumOfTheNumbers = 0;
+numbers.forEach(numbers =>
```

Comment:
```suggestion
numbers.forEach(number =>
```
он как итератор в единственном числе, следи за этим чтобы не запутаться